src/test/java/io/crate/jmx/integrationtests/V480MetricsITest.java

```diff
@@ -30,4 +32,14 @@ public class V480MetricsITest extends MetricsITest {
     String getCrateDistributionURL() {
         return URL;
     }
+
+    @Override
+    @Test
+    public void testConnectionsMetrics() {
```

Comment:
There is no way to get the cluster version from JMX or ` testCluster`, so to make the older versions run, need to override the test method to only check for the old stats.